.gitignore

```diff
@@ -1,5 +1,8 @@
 Thumbs.db
 .DS_Store
+.gradle
+.settings
+.project
 
```

Comment:
Reason to make this change now?

Reply:
After I upgraded VS code this files came in without changing anything. so just added it here

Reply:
Reset to previous version. However, if anyone faces the same issue we need to modify it!